### PR TITLE
Re-order mergeable pull requests based on priority

### PR DESCRIPTION
**Pull request by 50kudos**
~~First priority is for unchanged sha branch~~
~~Second priority is for changed sha branch, but working branch~~
~~Third priority is for working branch~~

### CODE REVIEW

File: lib/flash_flow/data/collection.rb [hunk 1]
```diff
@@ -65,6 +65,7 @@ def reverse_merge(old)
             branch.created_at = info.created_at
             branch.resolutions = info.resolutions.to_h.merge(branch.resolutions.to_h)
             branch.stories = info.stories.to_a | merged_branches[full_ref].stories.to_a
+            branch.merge_order ||= info.merge_order
```
Comment (bradleyjucsc) [inline lib/flash_flow/data/collection.rb:68]: This doesn't seem right anymore since you're initializing the merge order with zero.

Comment (50kudos) [inline lib/flash_flow/data/collection.rb:68]: I'll change the initialized value to `nil`, does that make sense?

Comment (bradleyjucsc) [inline lib/flash_flow/data/collection.rb:68]: I think so. So you don't even have to initialize it at all then.

**Comment by bradleyjucsc** [comment]
:+1:

**Pull request merged**